tests/UniswapAnchoredViewTest.js

```diff
@@ -327,13 +327,21 @@ describe('UniswapAnchoredView', () => {
 
   describe('constructor', () => {
 
-    it('should fail if anchor mantissa is too high', async () => {
+    it('should prevent bounds from under/overflow', async () => {
       const reporter = web3.eth.accounts.privateKeyToAccount('0x177ee777e72b8c042e05ef41d1db0f17f1fcb0e8150b37cfad6993e4373bdf10');
       const priceData = await deploy('OpenOraclePriceData', []);
-      const anchorMantissa = numToHex(2e18);
-      await expect(
-        deploy('UniswapAnchoredView', [priceData._address, reporter.address, anchorMantissa, 30, []])
-      ).rejects.toRevert("revert anchor tolerance is too high");
+      const anchorPeriod = 30, configs = [];
+      const UINT256_MAX = (1n<<256n) - 1n, exp = (a, b) => BigInt(a) * 10n**BigInt(b);
+
+      const anchorMantissa1 = exp(100, 16);
```

Comment:
Cool, I was going to ask for a case for `= 100e16` since it's the corner case. Looks great.